### DISCUSSION TITLE
Can't seem to get anything to run

### DISCUSSION THREAD

**Discussion by vpansare1** [dougransom/vix_utils#29] [Q&A]
Hi all

I installed the vix_utils package using pip install

I am now trying to run the programs in pycharm

I seem to be doing something wrong because I get error messages.  My goal is to simply track the VIX premium which is defined as VX30 - VIX

Could anyone give a basic recipe for how you set up this package, and what you run and in what order?

**Comment by dougransom** [comment]
Hi,

Can you try the command line program `vixutil` and see if that works for you?

You can view the examples, they are mentioned here:

https://github.com/dougransom/vix_utils

They are installed so you can browse to vix_utils/examples wherehever vix_utils is installed.

**Reply by vpansare1** [reply]
OK I got it working now thanks!

**Comment by dougransom** [comment]
Attached is a Jupyter notebook showing how to calculate the basis (what you referred to as premium).  You will have to trim it down as i have also computed the basis against vix9d etc to explore any correlations, but that is likely meaningless.  You will have to rename it to the appropriate extension.

[vx1_prep - Copy.txt](https://github.com/dougransom/vix_utils/files/12361178/vx1_prep.-.Copy.txt)

**Reply by vpansare1** [reply]
Thanks for providing this!  It seems there is an issue with installing / importing vectorbt - is it only compatible with python versions up to 3.11?  I just upgraded my python version to 3.12.  May need to wait for it to catch up

**Reply by vpansare1** [reply]
Also getting an error here:
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[9], line 2
      1 #manipulate the data to get what we want to display
----> 2 vix_spot_wide=vix_utils.pivot_spot_term_structure_on_symbol(vix_spot)
      3 vix_futures_monthly_skinny=vix_utils.select_monthly_futures(vix_futures)
      4 vix_futures_wide=vix_utils.pivot_futures_on_monthly_tenor(vix_futures_monthly_skinny)

AttributeError: module 'vix_utils' has no attribute 'pivot_spot_term_structure_on_symbol'

**Reply by vpansare1** [reply]
Is there an alternative to vectorbt than can be used that is compatible with python 3.12?

**Reply by vpansare1** [reply]
@dougransom thanks for your help with these issues!
